apps/nrm/views/partials/maincontent-left.html

```diff
@@ -1,7 +1,6 @@
 {{$content-outer}}
 {{$preloader}}{{/preloader}}
   <div class="column-{{columnWidth}}{{^columnWidth}}two-thirds{{/columnWidth}}">
-    <q>column-{{columnWidth}}{{^columnWidth}}two-thirds{{/columnWidth}}</q>
```

Comment:
Nice one

Reply:
I actually saw this and wrote a comment in my review asking why q tag.  I foolishly assumed it was to do with GDS weird template.  

I've learnt my lesson, just ask the damn question or check the GDS template the dev is overriding